seacrowd/sea_datasets/vimqa/vimqa.py

```diff
@@ -0,0 +1,187 @@
+                {
+                    "_id": datasets.Value("string"),
+                    "question": datasets.Value("string"),
+                    "answer": datasets.Value("string"),
+                    "type": datasets.Value("string"),
+                    "supporting_facts": datasets.features.Sequence(
+                        {
+                            "title": datasets.Value("string"),
+                            "sent_id": datasets.Value("int32"),
+                        }
+                    ),
+                    "context": datasets.features.Sequence(
+                        {
+                            "title": datasets.Value("string"),
+                            "sentences": datasets.features.Sequence(datasets.Value("string")),
+                        }
+                    ),
+                }
+            )
+        else:
+            features = schemas.qa_features
+            features["meta"] = {
+                "supporting_facts": datasets.features.Sequence(
+                    {
+                        "title": datasets.Value("string"),
+                        "sent_id": datasets.Value("int32"),
+                    }
+                ),
+                "context": datasets.features.Sequence(
+                    {
+                        "title": datasets.Value("string"),
+                        "sentences": datasets.features.Sequence(datasets.Value("string")),
+                    }
+                ),
+            }
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> list[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+        if self.config.data_dir is None:
+            raise ValueError("This is a local dataset. Please pass the data_dir kwarg to load_dataset.")
+        else:
+            data_dir = self.config.data_dir
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={"filepath": os.path.join(data_dir, "vimqa_train.json")},
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.VALIDATION,
+                gen_kwargs={"filepath": os.path.join(data_dir, "vimqa_dev.json")},
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.TEST,
+                gen_kwargs={"filepath": os.path.join(data_dir, "vimqa_test.json")},
+            ),
+        ]
+
+    def _generate_examples(self, filepath: Path) -> tuple[int, dict]:
+        with open(filepath, "r", encoding="utf-8") as f:
+            data = json.load(f)
+        for i, item in enumerate(data):
+            if self.config.schema == "source":
+                yield i, {
+                    "_id": item["_id"],
```

Comment:
adjust the example generations from col_naming proposed changes above
```suggestion
                    "id": item["_id"],
```